.pipelines/build-and-push-images-tagged.yml

```diff
@@ -0,0 +1,52 @@
+# Azure DevOps Pipeline building rp images and pushing to int acr
+trigger: none
+pr: none
+
+variables:
+- template: vars.yml
+
+resources:
+  pipelines:
+  - pipeline: buildImages
+    source: E2E
+    trigger: true
+
+jobs:
+- job: Build_and_push_images
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/v2')
+  displayName: Build release
+  pool:
+    name: ARO-CI
+
+  steps:
+  - template: ./templates/template-checkout.yml
+  - template: ./templates/template-az-cli-login.yml
+    parameters:
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+
+
+  - template: ./templates/template-push-images-to-acr-tagged.yml
+    parameters:
+      rpImageACR: $(RP_IMAGE_ACR)
+
+  - template: ./templates/template-az-cli-logout.yml
+  - script: |
+      cp -a --parents aro "$(Build.ArtifactStagingDirectory)"
+    displayName: Copy artifacts
+  - task: PublishBuildArtifacts@1
+    displayName: Publish Artifacts
+    name: aro_deployer
+
+  - script: |
+      set -xe
+      MESSAGE="$(git for-each-ref refs/tags/$TAG --format='%(contents)')"
+      echo "$MESSAGE" > $(System.DefaultWorkingDirectory)/release.txt
```

Comment:
Extracts release summary from annotated tag. The summary is stored in the file, which is used as `releaseNotesFile` input for `GithubRelease@0` tasks. Task does not support other form of input.

Task also require `githubConnection`, which is service connection in ADO of type Github pointing to `ARO-RP` repository with write access.

Task also generates auto-changelog based on commit titles. To make it work nicely [conventional commits](https://conventionalcommits.org/en/v1.0.0/) are required.